RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -30,21 +30,27 @@ public AddTestModuleCommand(IVBE vbe, RubberduckParserState state, IGeneralConfi
         }
 
         private const string FolderAnnotation = "'@Folder(\"Tests\")\r\n";
-        private const string ModuleLateBinding = "Private Assert As Object\r\n";
-        private const string ModuleEarlyBinding = "Private Assert As New Rubberduck.{0}AssertClass\r\n";
+        private const string AssertLateBinding = "Private Assert As Object";
 
-        private const string TestModuleEmptyTemplate = "'@TestModule\r\n{0}\r\n{1}\r\n";
+        private const string ModuleFakesProvider = "Private Fakes As {0}";
+        private const string AssertEarlyBinding = "Private Assert As Rubberduck.{0}AssertClass";
```

Comment:
I'd prefer to see more consistency with how these declarations are handled. Both the `FakesProvider` and the `AssertClass` require the same identical early/late binding conditional logic; the `AssertClass` has an additional `PermissiveAssertClass` option, but if we have `Private Fakes As {0}` then we should have `Private Assert As {0}` too, with separate logic to determine whether early-bound assert is `Rubberduck.AssertClass` or `Rubberduck.PermissiveAssertClass`.